app/Http/Controllers/Web/ImportFileController.php

```diff
@@ -159,11 +159,11 @@ public function store(Request $request, $importType)
                 'Last name' => $request->input('last_name'),
                 'Opt-in to Partner SMS/robocall' => $request->input('sms_opt_in') ?: 'No',
                 'Opt-in to Partner email?' => $request->input('email_opt_in') ?: 'No',
-                'Phone' => $request->input('mobile'),
+                'Phone' => $request->input('phone'),
```

Comment:
These and the other input field name updates are per updates on the RTV end I'm assuming?

Reply:
These renames are on our end -- I renamed these inputs on our test webform to better match the RTV field names, which are the array keys here.

Reply:
Ohhh lol right right! It's right there below in the PR 🤦